docs/known-issues.md

```diff
@@ -8,9 +8,12 @@ Known issue = any major user-facing bug or lack of user-facing feature that:
+| Issue                                                                                                                                                                                                                                                                                                | Date added | Unity Ticket | Workaround?                                                           | Fixed? |
+|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------|--------------|-----------------------------------------------------------------------|--------|
+| The Unity editor randomly crashes on MacOS when using ECS preview packages. This is not caused by the SpatialOS GDK.                                                                                                                                                                                 | 2018/07/25 | 1064084      | Simply retry.                                                         | No     |
+| IL2CPP does not currently work on Mac as it crashes upon starting the game. This is due to a bug in Unity that does not initialize a system correctly, if one of the injected `IComponentData` contains a `long`.                                                                                    | 2018/08/31 | 1076596      | Use the Mono compiler.                                                | Yes    |
+| `bool` is not a [blittable type](https://docs.microsoft.com/en-us/dotnet/framework/interop/blittable-and-non-blittable-types), so will cause an error if used to store component data.                                                                                                               | 2018/09/17 | None         | Use `BlittableBool` instead.                                          | No     |
+| Accessing component data from an entity query will segfault, therefore requests where `ResultType` is `SnapshotResultType` will be dropped before sending.                                                                                                                                           | 2018/09/19 | None         | Don't send entity queries where `ResultType` is `SnapshotResultType`. | No     |
+| [Writer [Require] fields](content/gameobject/reading-and-writing-component-data.md#how-to-update-component-properties): Updating a component using `Writer.Send` will lead to the SpatialOS GDK updating all properties of the component even if you indicated to only partially update a component. | 2018/09/25 | None         | None                                                                  | No     |
```

Comment:
Suggest:

"When you use Writer.Send to update a component's properties, it causes the GDK to update _all_ the properties of the component, even if you have set it to only update _some_ of the properties. Note that GDK updates unchanged properties to the value they already have."

Reply:
Fixed.